nni/compression/pytorch/utils/counter.py

```diff
@@ -24,112 +237,102 @@ def count_flops_params(model: nn.Module, input_size, custom_ops=None, verbose=Tr
 
     Parameters
     ---------
-    model : nn.Module
+    model: nn.Module
         target model.
-    input_size: list, tuple
-        the input shape of data
+    x: tuple or tensor
```

Comment:
What if my module has tuple of tensor as input? for example, `forward(self, x, y)`

Reply:
@ultmaster  the code checks whether x[0] is a tensor if x is tuple

Reply:
The check was not considerate. Have been fixed now.